src/main/java/org/threadly/examples/features/PrioritySchedulerExample.java

```diff
@@ -0,0 +1,153 @@
+package org.threadly.examples.features;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import org.threadly.concurrent.future.FutureUtils;
+import org.threadly.concurrent.future.ListenableFuture;
+import org.threadly.concurrent.PriorityScheduler;
+import org.threadly.concurrent.TaskPriority;
+
+
+/**
+ * Provides a basic example of the PriorityScheduler feature.
+ * 
+ * @author sreid8 - Sean Reid
+ *
+ */
+public final class PrioritySchedulerExample {
+  
+  /** static reference to the futures created */ 
+  private static List<ListenableFuture<?>> statFutures;
+  
+  /**
+   * Provided a PriorityScheduler that contains some basic ideas
+   * @param executor -> a PriorityScheduler instance
+   * @return -> the PriorityScheduler with tasks added
+   */
+  public static PriorityScheduler addTasksToPriorityScheduler(PriorityScheduler executor) {
+    final int numThreads = Runtime.getRuntime().availableProcessors() * 2;
+    List<ListenableFuture<?>> futures = new ArrayList<ListenableFuture<?>>(numThreads);
+    
+    executor.scheduleWithFixedDelay(new Runnable() {
+      
+      @Override
+      public void run() {
+        /*
+         * This is the task that will be run on a schedule.
+         * Example use case: A recurring task
```

Comment:
Sorry for nit picking, I can fix this if you want.  Example use case "a recurring task" is not very informative.  I think "Example use case: regular cleanup/system maintenance task"